src/js/containers/account/visualizations/AccountTimeVisualizationContainer.jsx

```diff
@@ -153,8 +223,93 @@ export class AccountTimeVisualizationSectionContainer extends React.Component {
         const xSeries = [];
         const ySeries = [];
         const allY = [];
-        if (this.state.hasFilteredObligated) {
-            // In progress
+        const hasFilteredObligated = (this.props.reduxFilters.objectClass.count() > 0)
+            || (this.props.reduxFilters.programActivity.count() > 0);
+
+        if (hasFilteredObligated) {
+            if (this.state.visualizationPeriod === 'quarter') {
```

Comment:
Is there a way to combine the `hasFilteredObligated` true and false logic into one? It seems that both are doing nearly identical parsing with the exception of `true` calculating the `obligatedOther` value. Would it make more sense to combine the two sets of logic and just use the if-statement to determine whether or not to calculate `obligatedOther`?